### DISCUSSION TITLE
alpine can't build as openssl software

### DISCUSSION THREAD

**Discussion by nanfengnan1** [alpinejs/alpine#4344] [1. Help]
libc.so    libc.so.6
localhost:/opt/program/software/openssl# ldd /usr/glibc-compat/lib/libc.so.6
        /usr/glibc-compat/lib/ld-linux-x86-64.so.2 (0x7fd1c2563000)
        ld-linux-x86-64.so.2 => /usr/glibc-compat/lib//ld-linux-x86-64.so.2 (0x7fd1c2321000)
Error relocating /usr/glibc-compat/lib//ld-linux-x86-64.so.2: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37
Error relocating /usr/glibc-compat/lib/libc.so.6: _res: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: svc_max_pollfd: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: __ctype_toupper: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: loc1: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: rpc_createerr: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: __ctype32_toupper: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: loc2: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: __ctype_b: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: __key_encryptsession_pk_LOCAL: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: svcauthdes_stats: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: __ctype32_tolower: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: __ctype32_b: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: svc_fdset: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: __ctype_tolower: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: __key_decryptsession_pk_LOCAL: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: svc_pollfd: symbol not found
Error relocating /usr/glibc-compat/lib/libc.so.6: unsupported relocation type 37

**Comment by SimoTod** [comment]
@nanfengnan1 wrong repo. This is Alpine JS, a frontend library for web development. You were probably looking for https://github.com/alpinelinux
